SpeechResponder/Service/ScriptRecoveryService.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace EddiSpeechResponder.Service
+{
+	public class ScriptRecoveryService
+	{
+		static ScriptRecoveryService()
+		{
+			WorkingDirectory = Directory.GetCurrentDirectory();
```

Comment:
Should we use the same directory location as we use for configuration data (including personalities)?
i.e. `Utilities.Constants.DATA_DIR + "\personalities\"`

Reply:
I agree